src/components/OrderableList/OrderableList.stories.tsx

```diff
@@ -0,0 +1,128 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { Checkbox, CheckboxState } from "@components/Checkbox/Checkbox";
+import { useListData } from "@react-stately/data";
+import { ItemDropTarget } from "@react-types/shared";
+import { Meta, Story } from "@storybook/react";
+import React, { FC, useState } from "react";
+import { Button } from "@components/Button/Button";
+import { OrderableList } from "./OrderableList";
+import { merge } from "@utilities/merge";
+import { GridNode } from "@react-types/grid";
+import { DragProperties, OrderableListItem } from "./types";
+import { dragStoryStyles } from ".";
+
+const renderContent = (
+    { value, prevKey, nextKey }: GridNode<OrderableListItem>,
+    { componentDragState, isFocusVisible }: DragProperties,
+) => (
+    <div
+        className={merge([
+            "tw-break-word tw-border tw-border-solid tw-rounded tw-p-3",
+            dragStoryStyles[componentDragState],
+            isFocusVisible && "tw-bg-violet-20",
+            (nextKey === null || prevKey === null) && "tw-bg-green-20",
+        ])}
+    >
+        {value.content}
+        <hr className="tw-mt-3 tw-mb-2 tw-border-black-20 tw-bg-black-20" />
+        <div className="tw-flex tw-justify-between">
+            <span className="tw-bold">{isFocusVisible && "Im in keyboard focus"}</span>
+            <span>Drag State: {componentDragState}</span>
+        </div>
+    </div>
+);
+
+const List: FC = () => {
```

Comment:
Because there's only a single story, you could export it directly and remove the Default export below
```
import { OrderableList as OrderableListComponent  } from "./OrderableList";

export const OrderableList: Story = () => {
   ...
   <OrderableListComponent ... />
}
```

Reply:
done